team_members_and_roles.md

```diff
@@ -4,3 +4,4 @@
 |-----|--------------|-----|
 |Ahmed Shahwan | [@shahwan42](https://github.com/shahwan42) | TeamLead/Web Developer |
 |Ziad Mohamed  | [@ziadnabil](https://github.com/ziadnabil) | Web Developer/Machine learning |
+|Mohamed Abd El-Sattar | [@mohamed-sattar](https://github.com/mohamed-sattar) | Android Teamleader/Android Developer |
```

Comment:
update it to `Mobile Team Lead/Mobile Developer`
we're doing RN for both iOS and Android :))

Reply:
ya sorry about that "D